### ISSUE TITLE
Select: redundant elements

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#2419]
We are observing an accessibility related issue with the Background color pop up option when navigated using screen reader.

Associated component is `<Select>`

**Summary:** 
Screen reader users rely on efficient and predictable navigation. When extra or redundant navigation occurs after interacting with "Background color" list box popup, users may have to navigate through unnecessary elements, leading to confusion and increased effort, screen readers (JAWS, VoiceOver) are additionally announcing “Background color list collapsed”, which feels redundant and impacts navigation clarity.

**Repro Steps :**
Defect : https://infoseci.atlassian.net/browse/AAGL-2189
Open URL: https://stage.infosec-iq.com/Account/settings/Branding
Activate create new brand control.
1. Navigate to the Background color list box control
2. Observe the navigation behavior when trying to navigate to the next logical element after the popup

**Actual Result**
The screen reader requires extra navigation steps beyond the Background color list box popup due to redundant or unnecessary elements.

**Expected Result**
Navigation beyond the Background color list box popup should be smooth, without unnecessary navigation stops.
WCAG Guidelines
WCAG 2.4.3 – Focus Order (Level A)
WCAG 1.3.1 – Info and Relationships (Level A)

**Test Environment:**
Environment: macOS | Chrome Version 146.0.7680.165 | VoiceOver
Windows 11 | Chrome Version 147.0.7683.0 | Jaws 2024

_Issue can be reproduced when you navigate using "control + option + right/left arrow key" in macOS_

[Details](https://cengage.slack.com/archives/C8Y4FR63U/p1775220538125939)
